Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -55,7 +60,20 @@ class ClipboardBarDisplayHandler {
     func checkIfShouldDisplayBar() {
         if shouldDisplayBar() {
             if let absoluteString = UIPasteboard.general.copiedURL?.absoluteString {
-                delegate?.shouldDisplayClipboardBar(absoluteString: absoluteString)
+                
```

Comment:
You should try to use guard where you can. it'll avoid having to indent too much code. Try 
``` guard let absoluteString = UIPasteboard.general.copiedURL?.absoluteString, shouldDisplayBar() else {  
     return  
}